shell/rapi2/Makefile

```diff
@@ -0,0 +1,91 @@
+LOCAL_PATH := $(call my-dir)
+FOR_LINUX :=1
+WEBUI :=1
+#NO_REND := 1
+#NO_X11 := 1
+RZDCY_SRC_DIR = ../../core
+
+include $(RZDCY_SRC_DIR)/core.mk
+
+#CC_PREFIX ?= arm-none-linux-gnueabi-
+
+CXX=${CC_PREFIX}g++
+CC=${CC_PREFIX}gcc
+AS=${CC_PREFIX}as
+STRIP=${CC_PREFIX}strip
+
+LD=${CC}
+
+MFLAGS := -marm -march=armv7-a -mtune=cortex-a9 -mfpu=neon -mfloat-abi=hard -funroll-loops
+ASFLAGS := -march=armv7-a -mfpu=neon -mfloat-abi=softfp
+
+LDFLAGS	:=	-g -Wl,-Map,$(notdir $@).map,--gc-sections -Wl,-O3 -Wl,--sort-common 
+
+
+CXXFLAGS := -g -O3  -D RELEASE -c -D TARGET_BEAGLE -D TARGET_LINUX_ARMELv7  
```

Comment:
I haven't touched the RPi Makefile, but yes, generally it would require -DUSE_VMEM_FILE and -DTARGET_RPI